docker-multinode/test/util.sh

```diff
@@ -64,7 +66,7 @@ start_tests() {
 
   # Compile minimal number of packages to run tests
   apt-get update
-  apt-get install -y make gcc
+  apt-get install -y build-essential
   cd ${K8S_LOCATION_PATH}
   go get -u github.com/jteeuwen/go-bindata/go-bindata
   make all WHAT=cmd/kubectl
```

Comment:
First thing. It doesn't work:
```
build/run.sh make WHAT="cmd/kubectl vendor/github.com/onsi/ginkgo/ginkgo test/e2e/e2e.test"
+++ [0829 09:39:13] Verifying Prerequisites....
+++ [0829 09:39:19] Building Docker image kube-build:build-d01156ba92.
+++ [0829 09:40:19] Running build command....
Error: No such image, container or task: kube-build-data-d01156ba92
+++ [0829 09:40:19] Creating data container
make: *** No targets specified and no makefile found.  Stop.
!!! Error in build/../build/common.sh:620
  '"${docker_cmd[@]}" "$@"' exited with status 2
Call stack:
  1: build/../build/common.sh:620 kube::build::run_build_command(...)
  2: build/run.sh:30 main(...)
Exiting with status 1
```
Second thing. In my opinion pure compilation only packages which are required to run test has smaller impact then using Docker. For me using Docker is overhead for this purpose.